drivers/Makefile

```diff
@@ -55,6 +55,7 @@ include eeprom/Make.defs
 include efuse/Make.defs
 include net/Make.defs
 include note/Make.defs
+include pcnt/Make.defs
```

Comment:
What is the reason for keeping a separate folder for pcnt and not putting it in drivers/timers?

Reply:
I thought driver is not related to timer. I tried to implement it like `rmt` driver

Reply:
I wasn't aware of the `rmt` driver. But now I see that it's also in the wrong place :) it should be in `drivers/rc`. What's more it looks like rmt do something similar like `drivers/rc/lirc_dev.c`

Reply:
if `qencoder` is in `timers` then pulse counter should also be in `timers`. I don't know how ESP implements this peripheral, but other vendors implement similar functionality using hardware timers.

Reply:
Are you sure about qencoder? Seems it is in [`drivers/sensors`](https://github.com/apache/nuttx/blob/master/drivers/sensors/qencoder.c).

Reply:
ups, my bad. `qencoder` is actually in `sensors` so this argument is invalid. But still, pulse counters are most likely implemented with timer peripheral.

Reply:
Timers and counters are basically the same thing. The difference is that a timer is clocked by an internal fixed-clock, while a counter is clocked by an external signal (external pulses). If a user is looking for a counter driver, most likely the first place they will look is `drivers/timers`

Reply:
Up to you but we can keep it for now because this driver will be there for temporary.